### DISCUSSION TITLE
won't save settings

### DISCUSSION THREAD

**Discussion by ouron35** [X2CommunityCore/xcom2-launcher#324] [Help]
AML Version: 1.5.0

Description
when i launch AML it won't find the game directory and mod folder, so i go to settings and find manually but after that i tried to "save settings" and appears "could not find file C:\Users\users\Documents\my game\XCOM2\XcomGame\Config" so it won't save the settings.
my game is on another hard drive not my main SSD

Attachments

[AML.log](https://github.com/X2CommunityCore/xcom2-launcher/files/8924604/AML.log)

**Comment by Iridar** [comment]
Have you tried launching AML as admin? Do you have any anti virus software? It might be interfering with AML writing to its own config file.

**Comment by RevZero** [comment]
As Iridar pointed out, something prevents AML from accessing certain folders/files. May also be a file system issue.

`System.IO.FileNotFoundException: Could not find file 'C:\Users\user\Documents\my games\XCOM2\XComGame\Config'.
File name: 'C:\Users\user\Documents\my games\XCOM2\XComGame\Config'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)`

**Comment by ouron35** [comment]
I tried  launching AML as admin and turning off my anti virus, still the same, i had a similar problem with another steam game a while ago but it was a compatibility with windows 10 and had to run as  windows 8, i tried the same with AML with no success.
the windows troubleshooter say its incompatibility but i don't think so.

**Comment by Dragon-32** [comment]
I wonder if the fact you've a user account called "user" is messing things up. Have you tried creating a differently named user account (e.g. ouron35) and trying to that?

**Comment by ouron35** [comment]
Dragon-32 it was a nice insight, i tried to change or if it had something, i saw that in "C:\Users\user\Documents\My Games\XCOM2\XComGame" i don't have the Config folder, so this is probably the problem because in the "C:\Users\user\Documents\My Games\XCOM2 War of the Chosen\XComGame" there are folder all.
does any one knows what i can do(i tried steam verifying the integrity and no prob)

**Comment by Dragon-32** [comment]
I've heard of people not having the My Games folders, for some reason they're not created.

I _think_ those people were manually able to create the folder structure and then the game would use them. My memory's a bit hazy, it was some comments on Steam from a while ago.
